packages/frontend/editor-ui/src/components/RunDataAi/utils.test.ts

```diff
@@ -180,3 +180,82 @@ describe(getTreeNodeData, () => {
 		]);
 	});
 });
+
+describe(createLogEntries, () => {
+	it('should return root node log entries in ascending order of executionIndex', () => {
+		const workflow = createTestWorkflowObject({
+			nodes: [
+				createTestNode({ name: 'A' }),
+				createTestNode({ name: 'B' }),
+				createTestNode({ name: 'C' }),
+			],
+			connections: {
+				B: { main: [[{ node: 'A', type: NodeConnectionTypes.Main, index: 0 }]] },
+				C: { main: [[{ node: 'B', type: NodeConnectionTypes.Main, index: 0 }]] },
+			},
+		});
+
+		expect(
+			createLogEntries(workflow, {
+				A: [
+					createTaskData({ startTime: +new Date('2025-04-04T00:00:00.000Z'), executionIndex: 0 }),
```

Comment:
we don't need to create a `Date` object, and then coerce it into a number.
```suggestion
					createTaskData({ startTime: Date.parse('2025-04-04T00:00:00.000Z'), executionIndex: 0 }),
```